### DISCUSSION TITLE
ecFSEOF Output Interpretation GECKO 3.1.3

### DISCUSSION THREAD

**Discussion by HossFir** [SysBioChalmers/GECKO#373] [GECKO3]
I've been working on simulations using GECKO 3.1.3 and the `ecFSEOF` script. It generated three TSV files containing information on `genes`, `reactions`, and `transporters`.

I'm curious about identifying potential candidates for up or down-regulation among the reactions and transporters, as all their slopes turned out to be positive.

In the genes TSV file, we categorized them as `KD`, `KO`, and `OE`. However, I'm uncertain how to interpret the other two files (reactions, and transporters). Can we simply rely on genes to identify candidates for overexpression and knockdown?

Also, it would be nice if the `v_matrix` could be saved as an output, so we see how the potential reactions fluxes changed with increasing the enforced objective function!

**Comment by ae-tafur** [comment]
Reactions and transporters are reported as reference or informative data from where gene targets came from, since any metabolic manipulation is done on genes. The slope must be interpreted together to the category (KO, KD, OE) since it represents the how related the flux through one reaction is to the selected objetive. 

V_matrix is reported as output (fseof.v_matrix) in matlab but not printed as file
